frontend/packages/dev-console/integration-tests/features/monitoring/link-Kiali-ODC-3697.feature

```diff
@@ -0,0 +1,138 @@
+    And user clicks on the Install button
+    And user selects stable channel
+    And user selects default installation mode
+    And user selects automatic approval strategy
+    And user clicks on Install button
+    Then user will see a modal saying Jaeger Operator is installed
+
+
+@regression
+Scenario: Create New Project to link to Kiali Dashboard
+    Given user is on the Installed Operator page
+    When user clicks on the Projects dropdown
+    And user clicks on Create Project button
+    And user enters name of Project
+    And user clicks on Create button
+    Then user will see new project created in Projects dropdown
+
+
+@regression
+Scenario: Create Istio Service Mesh Control Plane instance
+    Given user is on the Installed Operator page
+    And user has selected the newly created project
+    When user clicks on the Istio Service Mesh Control Plane link under Provided APIs tab
+    And user clicks on ServiceMeshControlPlane button
+    And user clicks on Create button
+    Then user will see the newly APIs created named basic-installation
+
+
+@regression
+Scenario: Create Istio Service Mesh Member Roll instance
+    Given user is on the Installed Operator page
+    And user has selected the newly created project
+    When user clicks on Red Hat OpenShift Service Mesh Operator
+    And user clicks on All Instances tab
+    And user clicks on Create New dropdown
+    And user clicks Istio Service Mesh Member Roll
+    And user clicks on Create button
+    Then user will see the newly APIs created named default
+
+
+@regression
+Scenario: Create Istio Service Mesh Member
+    Given user is on the Installed Operator page
+    And user has selected the newly created project
+    When user clicks on Red Hat OpenShift Service Mesh Operator
+    And user clicks on All Instances tab
+    And user clicks on Create New dropdown
+    And user clicks Istio Service Mesh Member
+    And user clicks on Create button
+    Then user will see the newly added Istio Service Mesh Member
+
+
+@regression
+Scenario: Project Details Overview Page
+    Given user is on the Projects page
+    When user clicks on the newly created project
+    Then user will see the Service Mesh Enabled inside Details card
+    And user will see the Launcher card
+    And user will see Open on Kiali link inside Launcher card
+
+
+@regression
+Scenario: Project Details Page
+    Given user is on the Projects page
+    When user clicks on the newly created project
+    And user clicks on the Details page
+    Then user will see the Service Mesh Enabled
+    And user will see the Launcher section
+    And user will see Open on Kiali link inside Launcher section
+
+
+@regression
+Scenario: Kiali link in the Topology filterbar
+    Given user is at Developer perspective
+    And user has selected the newly created project
+    When user clicks on the Topology page
+    Then user will see Kiali link in the Topology filterbar
+
+
+@regression
+Scenario: Click on Kiali link
```

Comment:
Update scenario description as Kiali dashboard display or something like that
Action doesn't refer to scenario description